ts/components/wallet/paymentSummary/UpdatedPaymentSummaryComponent.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * This component will display the transaction details if updates on the amount are identified
+ */
+import { H1, H3, Icon, Right, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import I18n from "../../../i18n";
+import variables from "../../../theme/variables";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+  amount: string;
+  updatedAmount: string;
+  expireDate: string;
+  tranche: string;
+}>;
+
+const styles = StyleSheet.create({
+  padded: {
+    paddingRight: variables.contentPadding,
+    paddingLeft: variables.contentPadding
+  },
+
+  darkContent: {
+    backgroundColor: variables.brandDarkGray
+  },
+
+  white: {
+    color: variables.brandLight
+  },
+
+  whiteStrike: {
+    fontSize: variables.fontSizeBase * 1.25,
+    color: variables.brandLight,
+    textDecorationLine: "line-through",
+    textDecorationStyle: "solid"
+  },
+
+  toAlignCenter: {
+    flexDirection: "row",
+    alignItems: "baseline"
+  },
+
+  toAlignColumnstart: {
+    flexDirection: "column",
+    alignItems: "flex-start"
+  },
+
+  underlined: {
+    textDecorationLine: "underline",
+    color: variables.brandLight
+  }
+});
+
+export default class UpdatedPaymentSummaryComponent extends React.Component<
+  Props
+> {
+  public render(): React.ReactNode {
+    return (
+      <Grid style={[styles.darkContent, styles.padded]}>
+        <View spacer={true} large={true} />
+        <Row>
+          <H3 style={styles.white}>
+            {I18n.t("wallet.firstTransactionSummary.amount")}
+          </H3>
+          <Right>
+            <H1 style={styles.whiteStrike}>{this.props.amount}</H1>
+          </Right>
+        </Row>
+        <View spacer={true} />
+        <Row style={styles.toAlignCenter}>
+          <H3 style={styles.white}>
+            {I18n.t("wallet.firstTransactionSummary.updatedAmount")}
+          </H3>
+          <Text> {""} </Text>
```

Comment:
what's this empty text?

Reply:
it allow me to concatenate 2 strings having a space between them. Inserting a space at the end of the first one does not work.

Reply:
Dont't seem you are really concatenating strings. What i see is a Header (H1) with an Icon on the Right. If you need space between use padings/margins.